### ISSUE TITLE
训练metaedsr跑完一个epoch就不动了

### ISSUE THREAD

**Issue by duanyucat** [XuecaiHu/Meta-SR-Pytorch#35]
你好，谢谢你提供的代码！
我的pytorch版本是1.2.0，测试代码是跑通了。然后想训练metaedsr模型，用了两块GPU，但是它跑完一个epoch后就不动了，GPU利用率为0，想请问一下这是什么原因呀ε=(´ο｀*)))
[Epoch 1] Learning rate: 1.00e-4
[1600/16000] [L1: 50.8884] 20.6+507.0s
......
[16000/16000] [L1: 15.0077] 14.5+151.8s
 这样是对的吗？

**Comment by XuecaiHu** [comment]
似乎你的GPU有人在占用，导致了模型长时间不出结果

**Comment by duanyucat** [comment]
> 似乎你的GPU有人在占用，导致了模型长时间不出结果

但是我特地暂停了另一个程序，两块都空闲着用来训练这个程序的

**Comment by f611001120b** [comment]
您好，我也在测试这个代码。请问您测试的时候 n_GPU 设置的是多少呢，您测试的时候显卡型号是什么呢？

**Comment by LILIXIYA** [comment]
您好， 

请问您尝试换GAN loss了么？我目前想尝试在这个代码中使用GAN loss， 但是程序报错。谢谢 @duanyucat @XuecaiHu
